prometheus.yml

```diff
@@ -0,0 +1,10 @@
+global:
+  scrape_interval: 20s
+  evaluation_interval: 20s
```

Comment:
I think that the default is 1 minute. I am interested in trying to get somewhat better resolution data than that.